Documentation/ContentElements/Content/Index.rst

```diff
@@ -1,74 +1,198 @@
-.. include:: /Includes.rst.txt
+..  include:: /Includes.rst.txt
 
+..  _content-working:
 
-.. _content-working:
-
-====================
-Working with content
-====================
+================
+Managing content
+================
 
 ..  image:: /Images/ManualScreenshots/ContentElements/ManageContent.png
     :alt: Manage content elements in the TYPO3 backend
```

Comment:
This screenshot shows just the page module link from the left main menu and nothing else. It does not show any content element

Reply:
I would remove that picture completely. Further it hangs around in right corner of the text. Everything you explain will be explained in the new better picture below.

Reply:
On the content element overview page each card has a picture and I am reusing the same picture on the top left of each page for consistency. We could maybe find a better image for this chapter.

See https://docs.typo3.org/m/typo3/tutorial-editors/13.4/en-us/ContentElements/Index.html

Reply:
I replaced it with one of the context menu